src/main/resources/db/migration/V054__Add_jurisdiction_status_index_on_envelopes.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX envelopes_jurisdiction_status_idx ON envelopes(jurisdiction, status);
```

Comment:
which queries will use this index?

Reply:
Please, refer https://tools.hmcts.net/confluence/display/RBS/SQL+queries+for+envelopes+table